src/features/profile/MyProfile.tsx

```
@@ -106,12 +106,24 @@
+          >
             {myProfileData.url}
           </a>
         )}
       </Section2>
-      <Section2 title="자기소개서">{myProfileData?.introduction}</Section2>
+      <Section2 title="자기소개서">
+        {myProfileData?.introduction ? (
+          <span
+            dangerouslySetInnerHTML={{
+              __html: myProfileData.introduction.replace(/\n/g, '<br />'),
+            }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)